tools/csv-merger/csv-merger.go

```diff
@@ -275,6 +275,10 @@ func unmarshalCSV(filePath string) *csvv1.ClusterServiceVersion {
 					},
 				},
 			},
+			{
+				Name:  "CSI_DRIVER_NAME_PREFIX",
+				Value: "openshift-storage",
```

Comment:
Since we've added flexibility to deploy ODF in any namespace and not only "openshift-storage", this might be confusing. Should we pick another static name instead like `odf` or something generic?

Reply:
This was the name agreed earlier (internal discussion) as we need to be backward compatibility and names cannot be changed on upgraded clusters and we are using the same name for ocs-client-operator as well. More details are in EPIC https://issues.redhat.com/browse/RHSTOR-5292